### PR TITLE
feat(Amplitude): Add isP2P attribute to the transport.stats analytics event

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

**Comment by hristoterezov** [comment]
Don't we already have this as user prop?

**Comment by jallamsetty1** [comment]
@hristoterezov, how do we set the user props ? Are these set using Statistics.analytics.addPermanentProperties() ? 
```
Statistics.analytics.addPermanentProperties({
            p2pFailed: false,
            forceJvb121: false
 });
```
Is that what you were referring to ?

**Comment by paweldomas** [comment]
> Don't we already have this as user prop?

No, we don't have it yet. Even if we would it might not be reliable to for the transport stats if those would be emitted while p2p is not yet established or if jvb transport is emitted after p2p has been established.

File: modules/statistics/AvgRTPStatsReporter.js [hunk 1]
```diff
@@ -941,6 +941,7 @@ export default class AvgRTPStatsReporter {
             return;
         }
         const transportStats = {
+            isP2P: data.transport[0].p2p,
```
Comment (paweldomas) [inline modules/statistics/AvgRTPStatsReporter.js]: can we keep the the p2p flag named `p2p` like other parts of the stats  report?

Comment (paweldomas) [reply modules/statistics/AvgRTPStatsReporter.js]: LGTM otherwise

Comment (jallamsetty1) [reply modules/statistics/AvgRTPStatsReporter.js]: ok sure

**Comment by hristoterezov** [comment]
> > Don't we already have this as user prop?
> 
> No, we don't have it yet. Even if we would it might not be reliable to for the transport stats if those would be emitted while p2p is not yet established or if jvb transport is emitted after p2p has been established.

@paweldomas Make sense!
@jallamsetty1 Please ignore my previous comment!

**Pull request merged**